backend/src/main/java/com/bakdata/conquery/metrics/ActiveUsersFilter.java

```diff
@@ -54,7 +54,7 @@ public void incrementPrimaryGroupCount(User user) {
 		if(primaryGroup.get().getMembers().size() <= ConqueryConfig.getInstance().getMetricsConfig().getGroupTrackingMinSize())
 			return;
 
-		SharedMetricRegistries.getDefault().counter(MetricRegistry.name(primaryGroup.get().getName(), "active")).inc();
```

Comment:
Ich würde immer noch Constanten in den Pfade der Metriken benutzen

Reply:
ich schau mal ob es das besser macht